pkg/ioutil/write_closer_test.go

```diff
@@ -47,3 +54,51 @@ func TestWriteCloseInformer(t *testing.T) {
 		assert.Fail(t, "write closer not closed")
 	}
 }
+
+func TestSerialWriteCloser(t *testing.T) {
+	const (
+		// Test 10 times to make sure it always pass.
+		testCount = 10
+
+		goroutine = 10
+		dataLen   = 100000 // larger than PIPE_BUF=4096
```

Comment:
Does the same test fail if non-serial writer is used?

Reply:
https://stackoverflow.com/questions/29981050/concurrent-writing-to-a-file

Reply:
It only has problem with old kernel. I can't reproduce this in my desktop. :)